src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -263,9 +263,20 @@ public View getPrimaryView() {
         return super.getPrimaryView();
     }
 
+    /**
+     * Creates a place holder view when there's no active branch indexed.
+     */
+    protected View getWelcomeView() {
+        return new MultiBranchProjectEmptyView(this);
```

Comment:
:bug: I think you meant `new OrganizationFolderEmptyView(this)`, which `getPrimaryView` should be made to use.